topic-3.2/wordWrappedTest.java

```diff
@@ -0,0 +1,50 @@
+package bootcamp32.topic3_2;
+
+import static org.junit.Assert.*;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.junit.Test;
+
+import bootcamp32.topic3_2.WordWrapping;
+
+public class wordWrappedTest {
```

Comment:
[High] Test class should be named "WordWrappedTest". Use uppercase for class naming.